### PR TITLE
Adding example of enhanced send message

### PR DESCRIPTION
**Pull request by lednax**
## Description of the change

- Added an example for sending a message to a `list`, a `list_pattern` and `user`

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,26 @@ resp = client.send_message(
...
+    message={'template': 'my-template', 'to': [
+      {
+        list_pattern: "<PATTERN>", // e.g. example.list.*
+      },
+      {
+        list_id: "<LIST_ID>", // e.g. your Courier List Id
+      },
+      {
+        email: "test@email.com"
+      }
```
Comment (tk26) [inline README.md:113]: I believe we need single quotes around keys. also comments in Ruby are `#`

**Comment by lednax** [comment]
Screen Shot 2022-03-07 at 8 03 25 PM
Screen Shot 2022-03-07 at 8 03 40 PM


@tk26

**Comment by lednax** [comment]
Screen Shot 2022-03-07 at 8 03 40 PM
Screen Shot 2022-03-07 at 8 03 25 PM

**Comment by lednax** [comment]
Screen Shot 2022-03-07 at 8 03 25 PM
Screen Shot 2022-03-07 at 8 03 40 PM

**Review by tk26 — APPROVED** [review]

**Pull request merged**